Added driver for the WeAct Studio 2.9in EPD (SSD1680)

EnricoRoss98 opened the pull request:
Added driver, demo and setup-example for the WeAct Studio 2.9in eInk display with an SSD1680 inside

peterhinch reviewed (commented):
This looks good but I don't understand the need for a specific demo. Is there a problem with the existing 2.9" demos on this display?

Other epaper drivers are in the drivers/epaper directory. I think we should stick to this arrangement.

EnricoRoss98 commented:
Ok, I moved the driver inside the folder drivers/epaper/ and removed the demo that I've previously added. In practice there's no need for a new dedicated demo: I wrote that demo for my needs and I thought that it would cost me nothing to add it to the project but you're right, is better to stay as light as possible and avoid clutter so I removed it.

peterhinch reviewed (approved):
Thank you.
